core/src/main/java/com/linecorp/armeria/internal/common/util/ChannelUtil.java

```diff
@@ -155,7 +155,7 @@ public final class ChannelUtil {
 
             try {
                 final Class<?> clazz = Class.forName(
-                        INCUBATOR_CHANNEL_PACKAGE_NAME + ".uring.IOUringChannelOption", false,
+                        CHANNEL_PACKAGE_NAME + ".uring.IoUringChannelOption", false,
```

Comment:
Netty io_uring module graduated from the incubator.
https://netty.io/news/2025/04/03/4-2-0.html#:~:text=io_uring%20support,class%20transport%20module.